package.json

```diff
@@ -62,8 +66,12 @@
     "passport-local": "^1.0.0",
     "react": "^0.14.2",
     "react-dom": "^0.14.2",
-    "react-helmet": "^1.0.1",
+    "react-helmet": "2.1.1",
```

Comment:
This is a good change. But some of the changes they've bumped up require us to have a little change in `base.html`

```javascript
<!doctype html>
<html lang="">

<head>
    TITLE

    META

    LINK

</head>

```